config/routes.rb

```diff
@@ -14,7 +14,7 @@
       get '/:token', to: 'sessions#create', as: 'create'
     end
 
-    resources :complete_curriculum_programmes, only: %i(index show)
+    resources :complete_curriculum_programmes, only: %i(index show), path: 'complete-curriculum-programmes'
```

Comment:
I wonder if there's a more general way to dasherise all paths 🤔

Reply:
Had a look, options looked a bit 'extreme' :sweat_smile: